package.json

```diff
@@ -15,12 +15,14 @@
     "start": "concurrently npm:start:*",
     "start:docs": "cd docs && npm run develop",
     "start:storybook": "start-storybook -p 6006",
+    "build:storybook": "build-storybook",
     "lint": "eslint '**/*.{js,ts,tsx,md,mdx}' --max-warnings=0",
     "lint:fix": "npm run lint -- --fix",
     "test": "jest",
     "test:update": "npm run test -- --updateSnapshot",
     "release": "npm run build && changeset publish",
-    "size": "size-limit"
+    "size": "size-limit",
+    "chromatic": "npx chromatic --project-token=91e12320417c"
```

Comment:
I know this looks scary. But this is the public project token, it's safe to commit this in code :)